components/mobile-scroll-lock/README.md

```diff
@@ -0,0 +1,21 @@
+MobileScrollLock
```

Comment:
I think that it can be renamed to `ScrollLock` as it is no longer dependent on any mobile related logic :)